src/libraries/System.Text.RegularExpressions/gen/UpgradeToGeneratedRegexCodeFixer.cs

```diff
@@ -276,7 +276,7 @@ static RegexOptions GetRegexOptionsFromArgument(ImmutableArray<IArgumentOperatio
                 }
                 else
                 {
-                    return generator.LiteralExpression(argument.Value.ConstantValue.Value);
+                    return argument.Value.Syntax;
```

Comment:
Per @joperezr comment, it's safe to reuse the syntax when the argument is a literal (ie, not `IInterpolationOperation`)

```suggestion
                    return argument.Value is ILiteralOperation literalOperation ? literalOption.Syntax : generator.LiteralExpression(argument.Value.ConstantValue.Value);
```

Reply:
Yes, I think this should work. So basically as long as the argument was a literal string, then we should fall back to calculate the constant value.